src/main/java/com/revlearn/team1/controller/DiscussionPostController.java

```diff
@@ -21,30 +22,32 @@
 @RequiredArgsConstructor
 public class DiscussionPostController {
 
-    private final DiscussionPostService disServ;
+    private final DiscussionPostServiceImp discussionPostService;
```

Comment:
It's better practice to import the interface instead of the class that implements it.